streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStoreTest.java

```diff
@@ -74,6 +74,7 @@ public void shouldOpenExistingStoreInRegularMode() throws Exception {
         rocksDBStore.init(context, rocksDBStore);
         assertThat(appender.getMessages(), hasItem("Opening store " + DB_NAME + " in regular mode"));
         LogCaptureAppender.unregister(appender);
+        rocksDBStore.close();
```

Comment:
this line was deleted by https://github.com/apache/kafka/commit/6ea3eedfd8c74d40157d6e489408d871e172ba34 and it causes trouble in reopening rocksDB due to lock error.